src/Sinch/Numbers/Numbers.cs

```diff
@@ -25,6 +35,42 @@ public interface ISinchNumbers
         ///     release numbers from projects, or list all numbers assigned to a project.
         /// </summary>
         public ISinchNumbersActive Active { get; }
```

Comment:
You may also mark this as obsolete before changing the visibility in the next major